src/fromager/commands/download_sequence.py

```diff
@@ -64,7 +64,7 @@ def download_one(entry: dict[str, typing.Any]):
 
         if entry["prebuilt"]:
             if include_wheels:
-                wheels.download_wheel(req, entry["source_url"], wkctx.wheels_downloads)
+                wheels.download_wheel(req, entry["source_url"], wkctx.wheels_prebuilt)
```

Comment:
I think we need it in both places, and we need to trigger rebuilding the local wheel index if we're running one. That way a prebuilt dependency of something else will be available when it's needed to build that thing.

Reply:
won't all of that happen on its own like you pointed out here: https://github.com/python-wheel-build/fromager/issues/390#issuecomment-2332694340

Reply:
Ah, right, OK.